tests/travis-ci/scripts/run_tests.sh

```diff
@@ -12,9 +12,6 @@ case "${TEST}" in
     PHPCodeSniffer)
         cd ${TRAVIS_BUILD_DIR}
         composer install
-        # This is a temporary hack until the tests for SPARQL Entity Storage
-        # module are enable din their project repo.
-        mv ./vendor/drupal/sparql_entity_storage ./modules/
```

Comment:
There's a PHPStan job line https://github.com/ec-europa/rdf_entity/blob/8.x-1.x/tests/travis-ci/scripts/run_tests.sh#L27 that also refers to sparql_entity_storage